Gulpfile.js

```diff
@@ -363,6 +362,14 @@ const eslint = (folder) => async () => {
         args.push("--fix");
     }
 
+    // Use stylish format on CI, so that it can be picked up by GH Action's rule matchers
+    if (cmdLineOptions.ci) {
+        args.push("--format", "stylish");
+    }
+    else {
+        args.push("--format", "autolinkable-stylish");
+    }
+

```

Comment:
Stylish:

```
/Users/ortatherox/dev/typescript/typescript-compiler/src/compiler/builderState.ts
  93:90  error  Missing semicolon  @typescript-eslint/semi
  94:83  error  Missing semicolon  @typescript-eslint/semi
```

Autolinkable Stylish

```
Error: /Users/ortatherox/dev/typescript/typescript-compiler/src/compiler/builderState.ts:93:90 @typescript-eslint/semi Missing semicolon.
Error: /Users/ortatherox/dev/typescript/typescript-compiler/src/compiler/builderState.ts:94:83 @typescript-eslint/semi Missing semicolon.
```

Stylish support is shipped [with setup-node](https://github.com/actions/setup-node) here: https://github.com/actions/setup-node/blob/master/.github/eslint-stylish.json

Reply:
Looks like this in the files tab of the PR:
![Screen Shot 2020-06-23 at 1 37 29 PM](https://user-images.githubusercontent.com/49038/85436245-b5cac080-b556-11ea-891d-2111f818b95e.png)